misc/docs/source/tutorial/code_snippets/stationxml_file_from_scratch.rst

```diff
@@ -0,0 +1,89 @@
+=============================================================================
+Creating a StationXML file from Scratch
+=============================================================================
+
+Creating a custom StationXML file is a task that sometimes comes up in
+seismology. This section demonstrates how to it with ObsPy. Please note that
+this is not necessarily easier or more obvious then directly editing an XML
+file but it does provider tighter integration with the rest of ObsPy and can
+guarantee a valid result at the end.
+
+Note that this assumes a certain familiarity with the `FDSN StationXML standard
+<https://www.fdsn.org/xml/station/>`_. We'll create a fairly simplistic
+StationXML file and many arguments are optional. ObsPy will validate the
+resulting StationXML file against its schema upon writing so the final file is
+assured to be valid against the StationXML schema.
+
+The following illustration shows the basic structure of ObsPy's internal
+representation.
+
+.. figure:: /_images/Inventory.png
+
+Each big box will be an object and all objects will have to be hierarchically
+linked to form a single :class:`~obspy.core.inventory.inventory.Inventory`
+object. An inventory can contain any number of
+:class:`~obspy.core.inventory.network.Network` objects, which in turn can
+contain any number of :class:`~obspy.core.inventory.station.Station` objects,
+which once again in turn can contain any number of
+:class:`~obspy.core.inventory.channel.Channel` objects. Instrument
```

Comment:
Good suggestion. New version is here: https://github.com/obspy/obspy/blob/4a43b69e0494432fa1b54fca8115f438df21c5ab/misc/docs/source/tutorial/code_snippets/stationxml_file_from_scratch.rst

I pushed directly to `master`.

At one point in the nearish future ObsPy should gain support for the NRL (see https://github.com/obspy/obspy/pull/1185) at which point the creation of fully usable stationxml files becomes feasible and we'll definitely write a tutorial for it.